saga/sagalogs/file_saga_log.go

```diff
@@ -0,0 +1,387 @@
+package sagalogs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/scootdev/scoot/saga"
+)
+
+// Writes Saga Log to file system.  Not durable beyond machine failure
+// Sagas are stored in a directory.  Each saga has a corresponding directory
```

Comment:
Sagas are more generic than our application.  Hence why they have our own library.  This in theory could be reused, its a pattern / protocol for Distributed Task Coordination.  

The JobDefinition / JobState is our application specific semantics.  Currently we only use ForwardRecovery Sagas.  Its analogous to a DB Schema and Application specific code.